gobblin-api/src/main/java/org/apache/gobblin/dataset/DatasetConstants.java

```diff
@@ -26,6 +26,8 @@ public class DatasetConstants {
   public static final String PLATFORM_SALESFORCE = "salesforce";
   public static final String PLATFORM_MYSQL = "mysql";
   public static final String PLATFORM_ICEBERG = "iceberg";
+  public static final String PLATFORM_CLUSTER = "cluster";
```

Comment:
this doesn't seem like the other ones here...
in what way are you defining a new 'platform' vs. just needing a string constant?